addons/web_editor/controllers/main.py

```diff
@@ -450,3 +451,15 @@ def reset_asset(self, url, bundle_xmlid):
                 be found
         """
         request.env['web_editor.assets'].reset_asset(url, bundle_xmlid)
+
+    @http.route("/web_editor/name_search_read", type="json", auth="user", website=True)
+    def name_search_read(self, model, name='', args=None, operator='ilike', limit=100, fields=None):
+        """
+        Performs a name_search followed by a search_read of the desired fields
+        """
+        if not request.env.user.has_group('website.group_website_designer'):
+            raise AccessError(_("You do not have access"))
+        Model = request.env[model]
+        records = Model._name_search(name, args, operator=operator, limit=limit)
+        ids = [record[0] for record in records]
+        return Model.browse(ids).read(fields)
```

Comment:
@JKE-be

Reply:
@JKE-be ;)

Reply:
@JKE-be What should we do with this task?

Reply:
lgtm

1. pas plus dangereux que le rpc call_kw 
2. strictement limité a group_website_designer

par contre pq  _name_search et pas name_search?

Reply:
We should probably limit to group_website_publisher and not designer. An HR manager should be able to edit a job position address.